### PR TITLE
added card game one night ultimate werewolf

### PR DESCRIPTION
**Pull request by tsomerve**
Submitting this PR means you have:

- [x] searched the [readme](../readme.md) file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [x] and will follow [the desired format](../formatting.md)

Do you have further questions? Check the [contribution guidelines](../contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -121,6 +121,17 @@
 | ------: | -------: | ------: |
 |   3 - 6 |       10 | 60-120m |
 
+### [One Night Werewolf](https://beziergames.com/products/one-night-ultimate-werewolf)
+
+> One Night Ultimate Werewolf is a fast-paced game where everyone gets to be a different role. In the course of only one night and the following morning, the players will determine who among them is a werewolf...hopefully.
+
+![werewolf game image](https://cdn.shopify.com/s/files/1/0740/4855/products/ONUW_3D_Box_2048x.png?v=1461752063)
```
Comment (edm00se) [inline readme.md]: The image here is huge, looks to be close to 1200px wide. Based on the link, it seems we could change this to have one that would be ~250px, as the formatting guide suggests.

```suggestion
![werewolf game image](https://cdn.shopify.com/s/files/1/0740/4855/products/ONUW_3D_Box_250x.png?v=1461752063)
```

Comment (tsomerve) [reply readme.md]: Thanks for your help! Had a lot of fun contributing :)

File: readme.md [hunk 2]
```diff
@@ -233,6 +233,16 @@
...
 
+### [One Night Werewolf](https://beziergames.com/products/one-night-ultimate-werewolf)
+
+> One Night Ultimate Werewolf is a fast-paced game where everyone gets to be a different role. In the course of only one night and the following morning, the players will determine who among them is a werewolf...hopefully.
+
+![werewolf game image](https://cdn.shopify.com/s/files/1/0740/4855/products/ONUW_3D_Box_250x.png?v=1461752063)
+
+| Players | Min. Age | Time    |
+| ------: | -------: | ------: |
+|   3 - 10|        8 |  10-15m |
```
Comment (edm00se) [inline readme.md]: The linter looks to have flagged this line due to the lack of padding in the first cell.

```suggestion
|   3 - 10 |       8 |  10-15m |
```

**Comment by tsomerve** [comment]
Thanks again!

**Comment by edm00se** [comment]
Thanks for contributing.

**Pull request merged**